hikari/impl/special_endpoints.py

```diff
@@ -1112,6 +1115,10 @@ def is_dm_enabled(self) -> undefined.UndefinedOr[bool]:
     def name(self) -> str:
         return self._name
 
+    @property
+    def name_localizations(self) -> typing.Mapping[typing.Union[locales.Locale, str], str]:
+        return self._name_localizations
+
```

Comment:
Again, would be nice to have them together :)

Reply:
So should this and `set_name_localizations` at the bottom together?

Reply:
Ideally this would be moved down with `set_name_localizations`